### ISSUE TITLE
Can remove bionics from NPCs without anaesthetic

### ISSUE THREAD

**Issue by Soadreqm** [Coolthulhu/Cataclysm-DDA#5580]
### Describe the bug

Removing bionics from NPCs with autodoc is possible even if there isn't enough anaesthetic. Anaesthetic is consumed if it is available, but the operation is still possible if there isn't enough.

Bionic removal should not be possible without sufficient anaesthetic.

### Steps To Reproduce

1. Have the Sensory Dulling CBM installed on the player
2. Recruit cyborg
3. Take cyborg to autodoc
4. Have several anaesthetic kits nearby, with small but nonzero amount of anaesthetic in one of them
5. Attempt to remove cyborg's malfunctioning bionics
6. Removal can be attempted, even if anaesthetic is insufficient for the operation

I didn't do any more testing, so I'm not sure how many of these steps are actually necessary.

### Screenshots

Note that "Available anesthetic" is less than "ANESTHETIC REQUIRED":
![cata-2024-10-16-21-35-42](https://github.com/user-attachments/assets/1fc25f0e-f486-4b06-b3c2-1ace439406a9)
![cata-2024-10-16-21-36-25](https://github.com/user-attachments/assets/fc6c242a-eb5a-486e-99cb-89e88cd8c8c4)

### Versions and configuration

- OS: Windows
    - OS Version: Windows 10 Pro N
- Game Version: 663ec9b67ec-dirty [64-bit]
- Graphics Version: Tiles
- LAPI Version: 2
- Game Language:  []
- Mods loaded: [
    Bright Nights [bn],
    Disable NPC Needs [no_npc_food],
    Simplified Nutrition [novitamins],
    No Rail Stations [No_Rail_Stations],
    Item Category Overhaul [Item_Category_Overhaul],
    Magiclysm [magiclysm],
    Bionic Slots [cbm_slots],
    Fuji's Military Profession Pack [fuji_mpp]
]

**Comment by Proxiehunter** [comment]
If you're doing it to an NPC maybe it _should_ remain possible if your character is a Psychopath, or with a moral penalty due to them screaming if your character _isn't_ a psychopath.

**Comment by Soadreqm** [comment]
Yeah, I don't think the autodoc couch has the kind of restraints that could stop a conscious patient from moving during surgery. If it did, there wouldn't really be any reason for the _player_ to need anaesthetic.

**Comment by chaosvolt** [comment]
> If you're doing it to an NPC maybe it _should_ remain possible if your character is a Psychopath

Average @RoyalFox2140 playthrough be like

But yeah, this seems like a bug more than anything else, possibly item identity stuff, or could be from active item and charges updates.

**Comment by RoyalFox2140** [comment]
That's... not how surgery works. We don't use anaesthesia just so patients are comfortable, even on organ donors anaesthesia is needed because of trauma to the body and the consequences of doing such things on an awake body.

Feels more a balance problem of course, anesthetic is a useful way to balance autodoc surgery.
